skactiveml/pool/_typi_clust.py

```diff
@@ -9,33 +9,32 @@
 import numpy as np
 
 from ..base import SingleAnnotatorPoolQueryStrategy
-from ..utils import MISSING_LABEL, labeled_indices
+from ..utils import MISSING_LABEL, labeled_indices, check_scalar
 from sklearn.cluster import KMeans
-from sklearn.base import ClusterMixin
 from sklearn.neighbors import NearestNeighbors
 
 
 class TypiClust(SingleAnnotatorPoolQueryStrategy):
-    """Typi Clust Selection
+    """Typical Clustering Selection
 
-    This class implements various Typi Cluster query strategies [1], which considers
-    both density and typicality of the samples.
+    This class implements Typical Clustering (TypiClust) query strategies [1], which considers
+    both diversity and typicality (representativeness) of the samples.
 
     Parameters
     ----------
     missing_label: scalar or string or np.nan or None, default=np.nan
         Value to represent a missing label
```

Comment:
```suggestion
        Value to represent a missing label.
```
Please add after each description of the parameter a dot (.), cf. suggestion above.